service/notesservice.php

```diff
@@ -222,9 +246,6 @@ private function getSafeTitleFromContent($content) {
             $title = $this->l10n->t('New note');
         }
 
-        // prevent directory traversal
-        $title = str_replace(array('/', '\\'), '',  $title);
```

Comment:
Did you look at the core code and how it is prevented there?
I would prefer if we wouldn't have to implement this ourselves...